docker/runner.Dockerfile

```diff
@@ -60,6 +60,31 @@ RUN mkdir -p /home/runner/.ssh \
 # Adjust permissions
 RUN chown -R runner:runner /home/runner
 
+# Setup podman
+ENV APT_KEY_DONT_WARN_ON_DANGEROUS_USAGE="1"
```

Comment:
you can probably just put this before apt-command you're doing instead of enabling it for the entire image

Reply:
Probably better yeah, added!